Neos.ContentRepository/Classes/Domain/Context/Dimension/ContentDimensionValueVariationEdge.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace Neos\ContentRepository\Domain\Context\Dimension;
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * The content dimension variation edge domain model
```

Comment:
still WITHIN one dimension -> one edge between specialization and generatization.

This Value Object is used when defining a content dimension (in its constructor). Never returned to the outside API.